class/nfs-subdir-external-provisioner.yml

```diff
@@ -10,12 +10,12 @@ parameters:
       - input_paths:
           - nfs-subdir-external-provisioner/component/app.jsonnet
         input_type: jsonnet
-        output_path: apps/
+        output_path: apps/${_instance}
```

Comment:
This is unnecessary and may cause ArgoCD to constantly redeploy the instances because there would be two identical ArgoCD apps with this change.

Reply:
I removed the unnecessary `_instance` identifier